Jenkinsfile_parameterized

```diff
@@ -0,0 +1,38 @@
+#!groovy
```

Comment:
can you update this file please see:
https://github.com/hmcts/cmc-citizen-frontend/blob/master/Jenkinsfile_parameterized